Blockchain/readme.md

```diff
@@ -52,10 +52,11 @@
 ## Solidity
 - [Remix IDE](https://remix-project.org/)
 - Solidity Tutorials :
-  - [1)Solidity With Javascript](https://youtu.be/gyMwXuJrbJQ)
-  - [2) Solidity With Python](https://youtu.be/M576WGiDBdQ) 
-  - [3) Smart Contractor Programme](https://www.youtube.com/channel/UCJWh7F3AFyQ_x01VKzr9eyA)
-  - [4) Solidity Tutorial](https://youtu.be/3g2WT2jms_k)
+- [1)Solidity With Javascript 10 hours course](https://youtu.be/cGQHXmCS94M)
+  - [2)Solidity With Javascript 31 hours course ](https://youtu.be/gyMwXuJrbJQ)
+  - [3) Solidity With Python](https://youtu.be/M576WGiDBdQ) 
+  - [4) Smart Contractor Programme](https://www.youtube.com/channel/UCJWh7F3AFyQ_x01VKzr9eyA)
```

Comment:
The python tutorial can be removed. Also please remove the smart contractor programme since it points to a YT channel. It can confuse folks as to what to actually learn.